README.md

```diff
@@ -52,6 +52,10 @@ to run in terminal / git :
 
```

Comment:
- [ ] Kindly edit your readme documentation to match this project. Your Readme has details about math-magicians and Tv-show project from JS. Kindly update with proper details related to this project.

- [ ] Remember to include instructions on how to clone the app and run the app locally.

Reply:
@julie-ify  thanks for reviewing.
my bad for not paying attention to my readme file, I fix it thanks again 
<img src="https://media0.giphy.com/media/3o6Zt6KHxJTbXCnSvu/giphy.gif"/>